data/shared/citizen/scripting/v8/timer.js

```diff
@@ -85,10 +85,18 @@
         animationFrames[animationFrames.length] = callback;
     }
 
-    function setInterval(callback, interval) {
+    function setInterval(callback, interval, ...argsForCallback) {
         const id = nextTimerId();
 
-        setTimer(id, callback, interval);
+	setTimer(
+            id,
+            function() {
+                try {
+                    callback(...argsForCallback);
+                }
```

Comment:
Missing catch or finally after try
```suggestion
                } catch { }
```